content/03.results.md

```diff
@@ -21,7 +19,7 @@ More recently, in 2016 and 2021 [@doi:10.1007/s00401-016-1545-1; @doi:10.1093/ne
 The Children’s Brain Tumor Tissue Consortium (CBTTC), currently the Children’s Brain Tumor Network (CBTN), opened its protocol for brain tumor and matched normal sample collection in 2011 and as such, the majority of the samples within the OpenPBTA dataset lack molecular subtype annotation. 
 In the absence of methylation arrays for tumor classification, we utilized key genomic features of entities described by the WHO in 2016, as well as Ryall and colleagues [@doi:10.1016/j.ccell.2020.03.011], coupled with clinician and pathologist review to subtype 64% (1,281/2,007) of tumor biospecimens with high confidence (**Supplemental Table S1**).
 Importantly, this collaborative molecular subtyping process allowed us to identify data entry errors (e.g., an ETMR entered as a medulloblastoma) and mis-identified specimens (e.g., Ewing sarcoma sample labeled as a craniopharyngioma), update diagnoses using modern terms (e.g., primitive neuro-ectodermal tumor [PNET] diagnoses), and discover rarer tumor entities within the OpenPBTA (e.g., H3-mutant ependymoma, meningioma with _YAP1::FAM118B_ fusion).
-**Table 1** {#tbl:subtypes} lists the subtypes we defined within the OpenPBTA, comprising of LGATs (N = 476), high-grade gliomas (N = 314), glialneuronal tumors (N = 14), medulloblastomas (N = 190), chordomas (N = 12), ependymomas (N = 65), other embryonal (N = 39), craniopharyngiomas (N = 51), neurocytoma (N = 6), gangliogliomas (N = 93), and Ewing sarcoma (N = 21). For methods, see **STAR Methods** and **Supplemental Figure {@fig:FigS1}S1**. 
+**Table 1** {#tbl:subtypes} lists the subtypes we defined within the OpenPBTA, comprising of LGATs (N = 476), high-grade gliomas (N = 314), glialneuronal tumors (N = 14), medulloblastomas (N = 190), chordomas (N = 12), ependymomas (N = 65), other embryonal (N = 39), craniopharyngiomas (N = 51), neurocytoma (N = 6), gangliogliomas (N = 93), and Ewing sarcoma (N = 21). For methods, see **STAR Methods** and **Figure S1**. 
```

Comment:
hmm seems we are both wrong, and this was likely written before the table was generated. I can update this to match the table. Above, we also wanted to remove the "To be classified" tumors because they were not actually subtyped. In addition, I was calculating based on biospecimen rather than tumor (sample_id), which I can also mention. 

```
subtypes <- read_tsv("~/Documents/GitHub/OpenPBTA-analysis/tables/results/Table1-molecular-subtypes.tsv")
hist <- unique(subtypes$broad_histology)
for (each in hist) {
  print(each)
select_subtypes <- subtypes %>%
 filter(broad_histology == each)
  print(sum(select_subtypes$n))
  }
[1] "Chordoma"
[1] 12
[1] "Diffuse astrocytic and oligodendroglial tumor"
[1] 314
[1] "Embryonal tumor"
[1] 229
[1] "Ependymal tumor"
[1] 65
[1] "Low-grade astrocytic tumor"
[1] 569
[1] "Mesenchymal non-meningothelial tumor"
[1] 21
[1] "Neuronal and mixed neuronal-glial tumor"
[1] 20
[1] "Tumors of sellar region"
[1] 51
```

Reply:
Ok, this is good. We have this record in case data changes later.